### ISSUE TITLE
Plugin getting disabled when opening a vault on Mobile (synced with a desktop)

### ISSUE THREAD

**Issue by damiankorcz** [dragonwocky/obsidian-tray#42]
As the title states the plugin automatically gets disabled when viewing a vault which is opened on Mobile. In my case, I use Syncthing to sync my vault between a Desktop (Windows), Laptop (Windows), Phone (Android) and Tablet (Android).

I know that the plugin doesn't really make sense for mobile, but it would be nice to keep it enabled but have it detect when it's loaded in mobile Obsidian and simply not do anything and remain enabled.

With the current behaviour if I open my vault on mobile, then later open my desktop or laptop, the vault won't open on boot since the plugin is disabled and was synced in that state.

Thanks!

**Comment by dragonwocky** [comment]
Hm, that's interesting. That's not intended behaviour, for sure.

I specified the plugin to be desktop only in the manifest, so what happens on mobile is entirely up to Obsidian (https://docs.obsidian.md/Reference/TypeScript+API/PluginManifest/isDesktopOnly), it's not something I'm handling manually. I suppose I could remove that option and implement my own "run without doing anything" logic specifically for mobile, but I wonder if this is maybe a bug in Obsidian itself that should be checked with their team as well?

**Comment by damiankorcz** [comment]
Yeah I'm not sure. I just tried it again and the part that makes Obsidian launch at boot seems to stick but I don't get the tray icon and the plugin is disabled. I'm guessing if the plugin is set to Desktop only then the behaviour is to turn it off on mobile just in case it would have untested results. Might be worth asking Obsidian devs for clarification.

Thanks for looking into this. Such a small plugin but it's very useful and saves me from reopening Obsidian through out the day :)

**Comment by dragonwocky** [comment]
I'm a little overloaded right now, but when I'm able to I'll see if I can get to the bottom of this properly. Thanks for the pickup!

**Comment by hockeymikey** [comment]
Do we have the plugin installed on mobile when we are seeing this? I did not have it installed on mobile and when remotely synced back to desktop it was disabled. Installing on mobile seemed to fix it but I also created a stub mobile solution on mobile which @dragonwocky threw out as an idea so not sure if its that plus the install that did the trick or just install. 

Can anyone confirm is .obsidian is being synced or not? I'm using nextcloud so I might have a double sync situation going. If we can disable the .obsidian sync in remotely sync and see what happens then we can start narrowing it down.
